spark_llm/prompt.py

```diff
@@ -119,3 +119,64 @@
 Assume the result of the Spark SQL query is stored in a dataframe named 'df', visualize the query result using plotly.
 There is no need to install any package with pip.
 """
+
+VERIFY_TEMPLATE = """
+Given 1) a PySpark dataframe, my_df, and 2) a description of expected properties, desc,
+generate a Python function to test whether the given dataframe satisfies the expected properties.
+Your generated function should take 1 parameter, df, and the return type should be a boolean.
+You will call the function, passing in my_df as the parameter, and return the output (True/False).
+
+In total, your output must contain ONLY the following code (no explanation words):
+1. Function definition, in Python
+2. Run the function with given dataframe, my_df, passed in as parameter
+3. Boolean indicating whether my_df satisfies the expectation described in desc (True/False)
+Your output MUST contain EACH of the above three things.
+
+For example:
+Input:
```

Comment:
Why do we need to mentiond how df is created?

Reply:
agreed, fixed